Bottom: 2b50ccfae9781b51735375585748c6dfd78afcd8
Top:    10e95770a5c358fa6c2bb2f471a2fcd66cb7a2dc
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-22 13:58:10 -0500

Refresh of all-normalize-vars

---
diff --git a/roles/docker-openldap/tasks/main.yml b/roles/docker-openldap/tasks/main.yml
index cda73dc..9a2b5a1 100644
--- a/roles/docker-openldap/tasks/main.yml
+++ b/roles/docker-openldap/tasks/main.yml
@@ -174,7 +174,7 @@
     state: started
 
 - name: set LDAP admin password
-  command: 'docker exec -i {{ openldap_container_name }} /docker-entrypoint.sh set_admin_pass {{ ldap_admin_password | quote }}'
+  command: 'docker exec -i {{ openldap_container_name }} /docker-entrypoint.sh set_admin_pass {{ openldap_admin_password | quote }}'
 
 - name: set LDAP proxyagent password
-  command: 'docker exec -i {{ openldap_container_name }} /docker-entrypoint.sh set_proxyagent_pass {{ ldap_admin_password | quote }} {{ ldap_proxyagent_password | quote }}'
+  command: 'docker exec -i {{ openldap_container_name }} /docker-entrypoint.sh set_proxyagent_pass {{ openldap_admin_password | quote }} {{ openldap_proxyagent_password | quote }}'
diff --git a/roles/docker-openldap/tasks/restore.yml b/roles/docker-openldap/tasks/restore.yml
index 00f6ef9..fd82c7f 100644
--- a/roles/docker-openldap/tasks/restore.yml
+++ b/roles/docker-openldap/tasks/restore.yml
@@ -20,7 +20,7 @@
 
 # - name: Set permissions for data-volume data
 #   file:
-#     path: '{{ host_openldap_docker_restore_dir }}/{{ item.ldif_file }}'
+#     path: '{{ openldap_docker_restore_dir }}/{{ item.ldif_file }}'
 #     state: touch
 #     owner: root
 #     group: root
